src/Slider/Slider.js

```diff
@@ -37,51 +37,133 @@ const valueInRangePropType = (props, propName, componentName) => {
   }
 };
 
+const getCrossAxisProperty = (axis) => ({
```

Comment:
Do we really need a function call here? Couldn't we save a function call by just using a hash?